db/migrate/20241231030009_remove_kit_id_from_orders.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveKitIdFromOrders < ActiveRecord::Migration[7.2]
+  def up
+    # define the changes kit id into product id
+    Order.where.not(kit_id: nil).find_each do |order|
+      order.update(product_id: order.kit_id, product_type: "Kit")
+  end
+  remove_column :orders, :kit_id, :integer
+  end
```

Comment:
Since we do not expect existing Order records as the app is not yet deployed, lets not include an update script in our migration.

```suggestion
  def change
    remove_column :orders, :kit_id, :integer
  end
```